openedx_webhooks/lib/edx_repo_tools_data/models.py

```diff
@@ -125,11 +117,18 @@ def agreement_expires_on(self):
         return expires_on
 
     @property
+    def before(self):
+        """
+        Optional[Dict[str, Dict]]: User's `before` data.
+        """
+        return self._data.get('before')
```

Comment:
In the past, the interface we needed to expose publically was, "what agreement was in place on a particular date".  "Agreement_expired" could be built on that, or alongside it.  I don't know of a time we needed "agreement_expires_on".